modules/share/main.lua

```diff
@@ -286,14 +286,14 @@ end
 -- Send player's ultimate and damage via map ping.
 -- This function must always be called from SendAttempt(), which checks whether it's safe to send data.
 local function SendData()
-	local pingType, ultType, ult, dmg, dps = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0
+	local pingType, ultType, ult, dmg, dps, lastPingTime = DAMAGE_UNKNOWN, ULT_HORN, 0, 0, 0, GetGameTimeMilliseconds()
 	local shareHorn = SV.enableUltimateShare ~= 0 and (hornSlotted and SV.enableUltimateShare or hasSaxhleel and type(SV.enableUltimateShare) == 'number' and SV.enableUltimateShare > 0)
 	local shareColos = SV.enableColosShare and colosSlotted
 	local shareAtronach = SV.enableAtronachShare and atronachSlotted
 	local shareMiscUltimates = SV.enableMiscUltimateShare
 	if M.IsEnabled() then
 		ultType = ULT_MISC
-		ult = roundDownToFive(zo_min(500, GetUnitPower("player", POWERTYPE_ULTIMATE)))
+		ult = roundDownToFive(zo_min(500, GetUnitPower("player", COMBAT_MECHANIC_FLAGS_ULTIMATE)))
```

Comment:
where does this flag come from? Where there API changes? i can not find it here: https://sir.insidi.at/or/apiexplorer/#q=COMBAT_MECHANIC_FLAGS_ULTIMATE

is the APIExplorer outdated ?